web/apps/labelstudio/src/pages/Settings/PredictionsSettings/PredictionsList.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import { formatDistanceToNow, parseISO } from 'date-fns';
+import { useEffect, useCallback, useContext } from 'react';
+import { FaEllipsisV } from 'react-icons/fa';
+import truncate from 'truncate-middle';
+import { LsEllipsis, IconPredictions, IconInfoOutline } from "../../../assets/icons";
+import { Button, Card, Dropdown, Menu, Checkbox, ToggleItems } from '../../../components';
+import { Tooltip } from '../../../components/Tooltip/Tooltip';
+import { DescriptionList } from '../../../components/DescriptionList/DescriptionList';
+import { confirm } from '../../../components/Modal/Modal';
+import { Oneof } from '../../../components/Oneof/Oneof';
+import { ApiContext } from '../../../providers/ApiProvider';
+import { Block, Elem, cn } from '../../../utils/bem';
+import { Form, Input, Label, TextArea, Toggle, Select } from '../../../components/Form';
+
+import './PredictionsList.styl';
+
+
+export const PredictionsList = ({ project, versions, fetchVersions }) => {
+  const api = useContext(ApiContext);   
+  
+  const onDelete = useCallback(async (version) => {
+    await api.callApi('deletePredictions', {
+      params: {
+        pk: project.id,        
+      },
+      body: {
+        model_version: version.model_version
+      }
+    });
+    await fetchVersions();
+  }, [fetchVersions, api]);
+  
+    return (
+        <div style={{ maxWidth: 680 }}>          
+          { versions.map(v => <VersionCard key={v.model_version}
+                                          version={v}
+                                          onDelete={onDelete} />) }
+      </div>
+  );
+};
+
+const VersionCard = ({ version, selected, onSelect, edittable, onDelete }) => {
+    const rootClass = cn("predictionCard");
+    const disabledStyle = {
+        opacity: edittable ? "1" : 0.3
+    };
+
+  const confirmDelete = useCallback((version) => {
+    confirm({
+      title: "Delete Predictions",
+      body: "This action cannot be undone. Are you sure?",
+      buttonLook: "destructive",
+      onOk(){ onDelete?.(version); },
+    });
+  }, [version, onDelete]);
+
+  
+    return (
+      <Block name="predictionCard">
```

Comment:
Just to keep consistent with the remainder of what I see in this PR. This is another job for a linter, as I have seen many takes on which way to go, which as long as we are consistent its correct 🙂.
```suggestion
      <Block name="prediction-card">
```